package.json

```diff
@@ -136,7 +136,7 @@
     "graphql": "^15.8.0",
     "graphql-subscriptions": "^2.0.0",
     "graphql-v14": "npm:graphql@14.7.0",
-    "graphql-v16": "npm:graphql@16.6.0",
+    "graphql-v16": "npm:graphql@16.8.1",
```

Comment:
Wondering what was the reason to leave `graphql` with version v15.
We usually put the latest version.